Project.toml

```
@@ -18,13 +18,13 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-AbstractGPs = "0.3"
-ChainRulesCore = "0.10"
+AbstractGPs = "0.3, 0.4, 0.5"
+ChainRulesCore = "0.10, 1"
```

```suggestion
ChainRulesCore = "1"
```
? consistent with AbstractGPs